src/platforms/node/common/sourcemaps/uploading/index.mdx

```diff
@@ -0,0 +1,7 @@
+---
+title: Uploading Source Maps
+---
+
+<PlatformContent includePath="sourcemaps/upload/primer/" />
```

Comment:
The structure here is different than the one for JS. JS has this content all one one page, but the Node content has the two pages listed separately. If it's possible to use the Node approach for both, I think that's better.

Reply:
👍